### PR TITLE
Fix local install script

### LINKED ISSUE #248: prepare-jenkins-builds param file not exists
**Describe the bug**
When running `prepare-jenkins-builds.sh`, the following code 

```
cd ${BASE_DIR}/ods-project-quickstarters/jenkins-slaves/maven/ocp-config
yes 'y' | tailor update -v --force
```

fails with error:

```
2019/11/19 10:13:19 Options could not be processed: Param directory ../../../../ods-configuration/ods-project-quickstarters/jenkins-slaves/maven/ocp-config does not exist
```

**To Reproduce**
1. Run prepare-jenkins-builds.sh via `vagrant ssh openshift -c "sudo /ods/ods-core/infrastructure-setup/scripts/prepare-jenkins-builds.sh"`

**Expected behavior**
Jenins slaves templats are configured and uploaded.

**Screenshots**

**Affected version:*
 - Version from master skript, installed by following the local installation guide.

**Additional context**

Same problem exists when running `vagrant ssh openshift -c "sudo /ods/ods-core/infrastructure-setup/scripts/prepare-jenkins-slave-builds.sh"`

```
Using project "cd".
Already on project "cd" on server "https://127.0.0.1:8443".
2019/11/19 11:53:39 Options could not be processed: Param directory ../../../../ods-configuration/ods-project-quickstarters/jenkins-slaves/nodejs8-angular/ocp-config does not exist
2019/11/19 11:53:39 Options could not be processed: Param directory ../../../../ods-configuration/ods-project-quickstarters/jenkins-slaves/nodejs10-angular/ocp-config does not exist
2019/11/19 11:53:39 Options could not be processed: Param directory ../../../../ods-configuration/ods-project-quickstarters/jenkins-slaves/python/ocp-config does not exist
2019/11/19 11:53:39 Options could not be processed: Param directory ../../../../ods-configuration/ods-project-quickstarters/jenkins-slaves/scala/ocp-config does not exist
Error from server (Forbidden): buildconfigs.build.openshift.io "jenkins-slave-python" is forbidden: buildconfigs.build.openshift.io "jenkins-slave-python" not found
Error from server (Forbidden): buildconfigs.build.openshift.io "jenkins-slave-scala" is forbidden: buildconfigs.build.openshift.io "jenkins-slave-scala" not found
Error from server (Forbidden): buildconfigs.build.openshift.io "jenkins-slave-nodejs8-angular" is forbidden: buildconfigs.build.openshift.io "jenkins-slave-nodejs8-angular" not found
Error from server (Forbidden): buildconfigs.build.openshift.io "jenkins-slave-nodejs10-angular" is forbidden: buildconfigs.build.openshift.io "jenkins-slave-nodejs10-angular" not found

```

### LINKED ISSUE #251: local installation scripts for version 2.0
**Describe the bug**
The scripts for local installation have to be corrected, so  that changes done in versin 2.0  are reflected.

**Additional context**
Installation guide has to be updated. See https://github.com/opendevstack/opendevstack.github.io/issues/125

### PR DESCRIPTION
**Pull request by stefanlack**
Work in Progress. fixes #251 fixes #248

### CODE REVIEW

**Comment by stefanlack** [comment]
Based on current state of master and branches of ods-quickstarters from https://github.com/kiwo/ods-quickstarters/tree/286/common and newest master of https://github.com/opendevstack/ods-quickstarters

File: infrastructure-setup/scripts/prepare-jenkins-builds.sh [hunk 1]
```diff
@@ -21,7 +21,7 @@ oc start-build -n cd jenkins-master --follow
 oc start-build -n cd jenkins-slave-base --follow
 oc start-build -n cd jenkins-webhook-proxy --follow
 
-cd ${BASE_DIR}/ods-project-quickstarters/jenkins-slaves/maven/ocp-config
+cd ${BASE_DIR}/ods-quickstarters/common/jenkins-slaves/maven/ocp-config
 yes 'y' | tailor update -v --force
 
-oc start-build -n cd jenkins-slave-maven --follow
+oc start-build -n cd jenkins-slave-maven --follow
```
Comment (michaelsauter) [inline infrastructure-setup/scripts/prepare-jenkins-builds.sh]: Why is the maven slave here at all? I'd assume we can remove it here. It is enough to have it listed in infrastructure-setup/scripts/prepare-jenkins-slave-builds.sh.

File: infrastructure-setup/scripts/prepare-sonarqube.sh [hunk 2]
```diff
@@ -40,7 +40,8 @@ echo ${TOKEN_BASE64}
 cd ${BASE_DIR}
 find . -iname "sonarqube.env" -exec sed -i "s|AUTH_TOKEN=$sonar_admin_password_base64|AUTH_TOKEN=$TOKEN_BASE64|g" {} \;
 find . -iname "templates.env" -exec sed -i "s|SONAR_SERVER_AUTH_TOKEN=$sonar_admin_password_base64|SONAR_SERVER_AUTH_TOKEN=$TOKEN_BASE64|g" {} \;
-find . -iname "secret.env" -exec sed -i "s|SONAR_SERVER_AUTH_TOKEN=$sonar_admin_password_base64|SONAR_SERVER_AUTH_TOKEN=$TOKEN_BASE64|g" {} \;
+find . -iname "secret.env"    -exec sed -i "s|SONAR_SERVER_AUTH_TOKEN=$sonar_admin_password_base64|SONAR_SERVER_AUTH_TOKEN=$TOKEN_BASE64|g" {} \;
+find . -iname "ods-core.env"    -exec sed -i "s|SONAR_AUTH_TOKEN_B64=$sonar_admin_password_base64|SONAR_AUTH_TOKEN_B64=$TOKEN_BASE64|g" {} \;
```
Comment (michaelsauter) [inline infrastructure-setup/scripts/prepare-sonarqube.sh]: I guess all references here that are not to `ods-core.env` do not apply anymore, right?

File: infrastructure-setup/scripts/prepare-jenkins-builds.sh (hunk 1 above)
Comment (stefanlack) [reply infrastructure-setup/scripts/prepare-jenkins-builds.sh]: @michaelsauter thank you for your comment. I have implemented it.

File: infrastructure-setup/scripts/prepare-sonarqube.sh (hunk 2 above)
Comment (stefanlack) [reply infrastructure-setup/scripts/prepare-sonarqube.sh]: @michaelsauter thank you for your comment. I have implemented it.

File: infrastructure-setup/ansible/inventories/dev/group_vars/all/version.yml [hunk 3]
```diff
@@ -6,4 +6,4 @@ atlassian_confluence_version: 6.14.3
 # set rundeck version
 rundeck_version: 2.11.8-1.59.GA
 git_version: 2.19.1
-tailor_version: 0.9.5
+tailor_version: 0.10.3
```
Comment (tjaeschke) [inline infrastructure-setup/ansible/inventories/dev/group_vars/all/version.yml]: We should use 0.10.4

Comment (stefanlack) [reply infrastructure-setup/ansible/inventories/dev/group_vars/all/version.yml]: Thank you!

**Pull request merged**